docs/config/io_helidon_webserver_WebServer.adoc

```diff
@@ -71,6 +71,22 @@ This is a standalone configuration type, prefix from configuration root: `server
 |`max-upgrade-content-length` |int |`65536` |Set a maximum length of the content of an upgrade request.
 
  Default is `64*1024`
+|`backpressure-buffer-size` |long |`5242880` |Set a maximum length of the unflushed response data sending buffer can keep without applying backpressure.
+Depends on `backpressure-policy` what happens if max buffer size is reached.
+
+ Default is `5*1024*1024` - 5Mb
+|`backpressure-policy` | String | `LINEAR` |Sets the strategy for applying backpressure to the reactive stream
+of response data.
+
+* LINEAR - Data chunks are requested one-by-one after previous data chunk has been written to Netty's buffer, when
```

Comment:
What are the use cases to support all these strategies? May be useful to include this info the doc as well.

Reply:
Speed vs memory as always, I wouldn't go that far with explaining use-cases, users who know how to work with backpressure on reactive pipeline have enough information, others shouldn't really touch it. 95% use-cases should fit in to the default setting.